monkey/infection_monkey/telemetry/attack/t1035_telem.py

```diff
@@ -0,0 +1,19 @@
+from infection_monkey.telemetry.attack.attack_telem import AttackTelem
+
+
+class T1035Telem(AttackTelem):
+    def __init__(self, status, usage):
+        """
+        T1035 telemetry.
+        :param status: ScanStatus of technique
+        :param usage: Usage string
+        """
+        super(T1035Telem, self).__init__('T1035', status)
+        self.usage = usage
```

Comment:
I thought about it for a while and decided it's important enough to change.
it doesn't make sense to send the full usage string as you're doing now, since there are very few possible usages (currently only from SMB, and only success/failure).
Change usage to be an enum. Currently use only 1 value for SMB.
When UI requests the info regarding this telemetry, THEN you should send the string (according to usage and status)

Reply:
I'll approve this PR. Fix this comment on PR #368